packages/issuer-token-endpoint/lib/IssuerTokenEndpoint.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  AccessTokenResponse,
+  Alg,
+  CNonceState,
+  CredentialOfferState,
+  getNumberOrUndefined,
+  IStateManager,
+  Jwt,
+  JWTSignerCallback,
+  Typ,
+} from '@sphereon/openid4vci-common'
+import express, { NextFunction, Request, Response, Router } from 'express'
+import { v4 } from 'uuid'
+
+const router = express.Router()
```

Comment:
What happened to the requirement that we should be able to pass in an existing express object?

Reply:
This is just a router to add flexibility on how the baseUrl is handled. The IssuerTokenEndpoint can be plugged on an existing express server **or** an express instance can be passed to IssuerTokenServer and they will work the same way.